### PR TITLE
f-restaurant-card@v0.15.0 - injectable wrapper components

### PR DESCRIPTION
**Pull request by jamieomaguire**
Note: No visual changes as a part of this PR.

### Added
- A renderless component to enable conditional dynamic components that do not render extra markup if the criteria is not met

### Changed
- Updated error boundary wrapping logic on card data points
- Used better semantic markup for restaurant tags
- Added styling override to prevent fozzie list styling from affecting restaurant tags in SearchWeb

### CODE REVIEW

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue [hunk 1]
```diff
@@ -49,5 +49,10 @@ export default {
     &:last-of-type {
         margin-right: 0;
     }
+
+    &:before {
```
Comment (sonnyprince) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue]: Is this an issue specific to searchweb? If so, should it live in search as apposed to this component (if its tech debt).

File: packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue [hunk 2]
```diff
@@ -174,9 +163,10 @@ export default {
         IconText,
         OfferIcon,
         LegendIcon,
-        RestaurantFees
+        RestaurantFees,
+        RenderlessSlotWrapper
     },
-    mixins: [ErrorBoundaryMixin],
+    mixins: [errorBoundaryProp],
     // NOTE: These are merely some placeholder props and not indicative of the props we will end up using
```
Comment (AlixRichards) [inline packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue:170]: I wonder if this name should stay as the original?
Looking in Fozzie most (although not all) mixins are named as you originally had it (PascalCase and ending with Mixin)

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue (hunk 1 above)
Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue]: in theory it's an issue for any legacy sites in the same position as searchweb using similar SCSS configuration @sonnyprince - happy to move it if you'd prefer

File: packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue (hunk 2 above)
Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue:170]: shall rename!

Comment (AlixRichards) [reply packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue:170]: Coolest 👍

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue (hunk 1 above)
Comment (sonnyprince) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue]: Probably best to move out, just incase its a recurring issue within the component. Atleast (worst case scenario) we could `restCard * {}` and catch all

Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantTags/RestaurantTags.vue]: shall do!

**Review by AlixRichards — APPROVED** [review]

**Review by dandel10n — APPROVED** [review]

**Pull request merged**